src/test/java/com/google/javascript/cl2dts/require/RequireTest.java

```diff
@@ -0,0 +1,53 @@
+package com.google.javascript.cl2dts.require;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.javascript.cl2dts.DeclarationGeneratorTests.JS;
+import static com.google.javascript.cl2dts.DeclarationGeneratorTests.getTestInputFiles;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import com.google.common.base.Function;
+import com.google.common.collect.FluentIterable;
+import com.google.common.io.Files;
+import com.google.javascript.cl2dts.DeclarationGenerator;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.PrintStream;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+
+// TODO(alexeagle): should we have an explicit test class like this in each directory?
+// Or should we auto-discover them as we do for test cases in the root directory?
```

Comment:
auto-discover usually doesn't scale well, when we need some flexibility. Rather go for explicit over implicit.